build.gradle.kts

```
@@ -7,22 +10,32 @@ import org.jetbrains.intellij.IntelliJPlugin
 import org.jetbrains.intellij.IntelliJPluginExtension
 import org.jetbrains.intellij.tasks.*
 import org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel.*
+import se.ascp.gradle.GradleVersionsFilterPlugin
 
 plugins {
   checkstyle
   `java-library`
   scala
+}
 
-  // See https://youtrack.jetbrains.com/issue/KTIJ-19369
-  // for the details of a false-positive error reported here by IntelliJ
-  alias(libs.plugins.taskTree)
-  alias(libs.plugins.versionsFilter)
-  alias(libs.plugins.versionCatalogUpdate)
-
-  alias(libs.plugins.grgit) apply false
-  alias(libs.plugins.jetbrains.grammarkit) apply false
+buildscript {
+  repositories {
+    gradlePluginPortal()
+  }
+  dependencies {
+    classpath(libs.pluginPackages.grgit)
+    classpath(libs.pluginPackages.jetbrains.grammarkit)
+    classpath(libs.pluginPackages.jetbrains.kotlin)
+    classpath(libs.pluginPackages.taskTree)
+    classpath(libs.pluginPackages.versionCatalogUpdate)
+    classpath(libs.pluginPackages.versionsFilter)
+  }
 }
 
+apply<VersionCatalogUpdatePlugin>()
+apply<TaskTreePlugin>()
+apply<GradleVersionsFilterPlugin>()
```

Nit:
```suggestion
apply<GradleVersionsFilterPlugin>()
apply<VersionCatalogUpdatePlugin>()
apply<TaskTreePlugin>()
```
Just for clarity, coz `VersionCatalogUpdatePlugin` depends on `GradleVersionsFilterPlugin`